paloesg/excide
# Update existing workflow's and workflow actions deadline template is updated.

# Description

Upon editing the template and saving it, if the workflow or workflow action deadline was changed, the existing incomplete workflows or workflow actions will also be updated to the new deadline date based on their creation date.

Notion link: https://www.notion.so/0b9be2495a23438995cc85fd0680affa?v=b8425151e89b4b14bfe425fc33d558b5&p=ac13ba2f29274bcdbca295d1ce6b129e

## Remarks

N.A.

# Testing

- Edit a template with existing workflows and workflow actions (ideally with different created dates)
- Change either the template deadline or the task deadline or both and save template
- Verify that in the template page, the deadlines of existing, incomplete workflows and workflow actions are updated accordingly.
